pom.xml

```diff
@@ -67,7 +67,7 @@
     <maven.version>3.9.9</maven.version>
 
     <flink.version>1.14.6</flink.version>
-    <spark.version>3.3.4</spark.version>
+    <spark.version>3.4.4</spark.version>
```

Comment:
The IDEA Maven plugin prioritizes the outer properties in the artifactId. 
Therefore, we need to update the version in the pom file when using the IDEA Maven plugin. 
This update does not impact the compilation process.

Reply:
The spark version format should be majorVersion.minorVersion.bugfixVersion

How about using `spark.binary.version` property with majorVersion.minorVersion?

Reply:
I am fine with it~